src/SFA.DAS.EmployerApprenticeshipsService.Application/Commands/UpdateApprenticeshipStatus/UpdateApprenticeshipStatusCommandHandler.cs

```diff
@@ -52,6 +60,9 @@ public UpdateApprenticeshipStatusCommandHandler(IEmployerCommitmentApi commitmen
             await ValidateDateOfChange(command, validationResult);
 
             await _commitmentsApi.PatchEmployerApprenticeship(command.EmployerAccountId, command.ApprenticeshipId, apprenticeshipSubmission);
+
+            var apprenticeship = await _commitmentsApi.GetEmployerApprenticeship(command.EmployerAccountId, command.ApprenticeshipId);
```

Comment:
We could check to see if SendEmail is enabled in config before fetching the apprenticeship, but as we always run with SendEmail enabled, it's probably not worth it!